sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,7 @@
+SELECT "genres"."name" as "genre",
+       count("filmGenre"."genreId")
+FROM "filmGenre"
+JOIN "genres" using ("genreId")
+JOIN "castMembers" using ("filmId")
+WHERE "actorId" = 178
```

Comment:
It's important that you can write queries that work with the limited information you have available. For example, here you only know the actor's full name up-front, not their id, so your query should use that information.

---

![image](https://user-images.githubusercontent.com/16143771/219540730-d976634f-9ecd-41fa-8e83-43d9cd18a81d.png)